pkg/kube/kubehttp.go

```diff
@@ -213,9 +213,12 @@ func (k *KubeHTTP) Apply(ctx context.Context, manifest []byte, namespace string)
 		return fmt.Errorf("failed to dynamic resource interface: %w", err)
 	}
 
+	force := true
```

Comment:
Always forcing this will cause all the manifests to be force patched on install.  I'd prefer that we only force the install for the application CRD.  I don't know if there would be issues force installing other manifests from flux.

Reply:
After discussing this a bit more, we are going to proceed this way (after some more testing) and look at a more complete upgrade process.